drivers/sd/bcm2836/rpisdhc/rpisdhc.cpp

```diff
@@ -1348,12 +1361,37 @@ NTSTATUS SDHC::sendCommandInternal (
 
         status = this->getLastCommandCompletionStatus();
         if (!NT_SUCCESS(status)) {
+            //
+            // Need to read status register again to get real error code
+            //
+            _HSTS hsts{ 0 };
+            this->readRegisterNoFence(&hsts);
+            //
+            // CMD1 alwary return CRC7 error on EMMC device
+            // Clear all ERROR mask set and return Success
+            // Temp clear error flags for read/write test command
+            //
+            if(((Cmd.Fields.Command == 0x1) && (hsts.Fields.Crc7Error))) //||
+                //(Cmd.Fields.Command == 0xe) ||
+                //(Cmd.Fields.Command == 0x13))
+               {
+                //
+                // CMD1 alwary return CRC7 error on EMMC device
+                // Clear all ERROR mask set and return Success
+                //
+                //hsts.AsUint32 |= _HSTS::UINT32_CRC7_ERROR_MASK;
```

Comment:
Can we remove the commented out code? Or is there a reason you are keeping it here?
Also, the comment above would seem to be incorrect now since you aren't clearing the ERROR mask anymore?

Reply:
Removed the commented code.
Because the status is read from register before this, ant the value is with this flag set, there is no need to set it again. It only needs to write this value to register to clear error status.

Reply:
Can you remove the commented code completely? I don't see a need to keep it here and commented out.